lib/sisimai/bite/email/google.rb

```diff
@@ -0,0 +1,320 @@
+module Sisimai::Bite::Email
+  module Google
+    # Sisimai::Bite::Email::Google parses a bounce email which created by Gmail.
+    # Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/Google.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
+        :from    => %r/[@]googlemail[.]com[>]?\z/,
+        :subject => %r/Delivery[ ]Status[ ]Notification/,
+      }
+      Re1 = {
+        :begin   => %r/Delivery to the following recipient/,
+        :start   => %r/Technical details of (?:permanent|temporary) failure:/,
+        :error   => %r/The error that the other server returned was:/,
+        :rfc822  => %r{\A(?:
+             -----[ ]Original[ ]message[ ]-----
+            |[ \t]*-----[ ]Message[ ]header[ ]follows[ ]-----
+            )\z
+        }x,
+        :endof   => %r/\A__END_OF_EMAIL_MESSAGE__\z/,
```

Comment:
__[rubocop]__
* Layout/SpaceAroundOperators: Operator `=>` should be surrounded by a single space. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)

<!-- Comment by [SideCI](https://sideci.com) -->